tests/platform_helper/domain/test_maintenance_page.py

```diff
@@ -330,11 +330,12 @@ def test_listener_roll_back_on_exception(
 
         excepted = (
             "Maintenance page failed to activate for the test-application application in environment development.\n"
-            f"Rolled-back rules: {expected_roll_back_message}\n"
             "Original exception: An error occurred (ValidationError) when calling the CreateRule operation: Simulated failure"
         )
         assert str(e.value).startswith(excepted)
 
+        # TODO add check for warn output or mock Io and assert called with
+
```

Comment:
Intentionally left in? Or is there a ticket to address this? If the latter it would be good to have the ticket number in here

Reply:
Kate caught this too, I used to test it in the message now I want to cehck that the warning log is correct. I will mock IO to test assert_called_with